content/docs/guides/autoscaling-guide.md

```diff
@@ -68,4 +68,32 @@ To edit a compute endpoint:
 
 ## Monitor autoscaling
 
-The `neon_utils` extension provides a `num_cpus()` function for monitoring how the _Autoscaling_ feature allocates compute resources in response to workload. For more information, see [The neon_utils extension](/docs/extensions/neon-utils).
+From the Neon Console, you can view how your CPU and compute hour usage scales over time (last hour, day, and week). From the **Branches** page, open the branch you want to inspect, then open the **Edit** modal for its compute endpoint.
+
+![autoscaling graph example](/docs/guides/autoscaling_graphs_sample.png)
+
+Some key points about this Autoscaling view:
+
+- Allocation refers to the vCPU and memory size provisioned to handle current demand; autoscaling automatically adjusts this allocation, increasing or decreasing the allocated vCPU and memory size in a step-wise fashion as demand fluctuates, within your minimum and maximum limits.
+- Your minimum and maximum limits are shown as solid white lines. This represents the allocation boundary: the size of your allocated vCPU/memory stays within this range so long as your compute endpoint remains active. It scales to zero after the defined period of inactivity.
```

Comment:
I would suggest to either refrain from noting specific colors or designate the theme

Reply:
maybe "solid horizontal lines"?

Reply:
Good call. done.